packages/react-components/src/components/ChatMessageComponent.tsx

```diff
@@ -34,61 +31,6 @@ type ChatMessageProps = {
   strings: MessageThreadStrings;
 };
 
-// https://stackoverflow.com/questions/28899298/extract-the-text-out-of-html-string-using-javascript
-const extractContent = (s: string): string => {
```

Comment:
Note: just moved these into a seperate file, no functional changes made to these helper functions